### PR TITLE
GEODE-10127: Use correct hostname in WAN with LB test

### PR DESCRIPTION
**Pull request by upthewaterspout**
This test is making gateway sender side members to connect to the receiver side
through a load balance.  This test used "localhost" as the hostname for
clients. That worked previously, because we were actually losing the
hostname-for-clients in the RemoteLocatorResponse, and instead sending the IP
address, due to the changes in GEODE-8955.

With the fix for GEODE-10127, we are now correctly sending the
hostname-for-clients to the gateway sender, and so
now "localhost" doesn't work. The correct hostname-for-clients is the IP
address we get from docker-compose.

Changing the test to pass the IP address instead of "localhost" to the receiver
locator, just like we do for the receivers themselves.

Re-enabling the test

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### CODE REVIEW

File: geode-assembly/src/acceptanceTest/java/org/apache/geode/cache/wan/SeveralGatewayReceiversWithSamePortAndHostnameForSendersTest.java [hunk 1]
```diff
@@ -101,8 +100,8 @@
   @BeforeClass
   public static void beforeClass() throws Exception {
     // Start locator
-    docker.execForService("locator", "gfsh", "run",
-        "--file=/geode/scripts/geode-starter-locator.gfsh");
+    docker.execForService("locator", "gfsh", "-e",
+        startLocatorCommand());
     // Start server1
     docker.execForService("server1", "gfsh", "run",
         "--file=/geode/scripts/geode-starter-server1.gfsh");
```
Comment (albertogpz) [inline geode-assembly/src/acceptanceTest/java/org/apache/geode/cache/wan/SeveralGatewayReceiversWithSamePortAndHostnameForSendersTest.java:107]: I suggest we get rid of the .gfsh file here and inline the command and the same for the next two gfsh invocations.

**Review by albertogpz — APPROVED** [review]

File: geode-assembly/src/acceptanceTest/java/org/apache/geode/cache/wan/SeveralGatewayReceiversWithSamePortAndHostnameForSendersTest.java (hunk 1 above)
Comment (upthewaterspout) [reply geode-assembly/src/acceptanceTest/java/org/apache/geode/cache/wan/SeveralGatewayReceiversWithSamePortAndHostnameForSendersTest.java]: Good idea! I turned all of the scripts into `gfsh -e` commands inline in the java code. Thanks!

**Comment by nabarunnag** [comment]
Dan can we still use hostnames for --hostname-for-clients or it has to be IP always. Is this change just only for Docker compose case

**Comment by upthewaterspout** [comment]
@nabarunnag  - In general hostname-for-clients *should* be a hostname, especially if you are using TLS. However, IP addresses should also work - we should just be passing whatever is in hostname-for-clients all the way to the client/gateway sender.

I'm not clear enough about networking with docker compose to know if there is a hostname we could use in this case instead of the IP address, but maybe there is something?

**Review by boglesby — APPROVED** [review]

**Pull request merged**